### ISSUE TITLE
US10: Speech Reader

### ISSUE THREAD

**Issue by voacado** [voacado/accessible-spreadsheet#10]
**As a visual impaired user, I want to be able to use this application using a screen reader application so that I am still able to use the software despite being visually impaired.**

Priority: Low
Estimate: 8 points

Acceptance Criteria:

- When the “Screen Reader” option in the Options Pane is toggled, a text-to-speech tool should be invoked that reads out the contents of a cell to the user through their speakers.
- The TTS tool should also try to describe things like images and charts.

**Comment by voacado** [comment]
Implemented (https://github.com/voacado/accessible-spreadsheet/commit/e8c0c5de788ddd9d7cdbe2d85ae7eb58b7d87e17)
